Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2927,20 +2925,8 @@ extension BrowserViewController: IntroViewControllerDelegate {
             let signInVC = FxAContentViewController(profile: profile)
             signInVC.delegate = self
             
-            // Append any passed query strings to the signin url. Note that you can't
-            // override the service and context params.
-            if fxaOptions.query != nil {
-                var url = "";
-                for param in fxaOptions.query! {
-                    if (param.key == "service" || param.key == "context") {
-                        continue;
-                    }
-                    url = url + "&" + param.key + "=" + param.value
-                }
-                let newUrl = URL(string: "\(profile.accountConfiguration.signInURL)\(url)")
-                signInVC.url = newUrl
-            } else {
-                signInVC.url = profile.accountConfiguration.signInURL
+            if fxaOptions != nil {
+                signInVC.fxaOptions = fxaOptions!
```

Comment:
Its a style thing. But its better to pass `fxaOptions` as an init variable than assign properties to a VC like this.